dependencies/default/pom.xml

```diff
@@ -51,7 +51,9 @@
     <governator-api.version>1.17.12</governator-api.version>
     <guava.version>31.1-jre</guava.version>
     <failureaccess.version>1.0.1</failureaccess.version>
-    <guice.version>4.2.0</guice.version>
+    <guice.version>5.1.0</guice.version>
+    <guice.extension.version>5.1.0</guice.extension.version>
+
```

Comment:
we sort by alphabet, I think we can remove this blankline.

Reply:
fixed